mllib/src/main/scala/org/apache/spark/ml/classification/DecisionTreeClassifier.scala

```diff
@@ -90,6 +92,9 @@ final class DecisionTreeClassifier(override val uid: String)
   override def copy(extra: ParamMap): DecisionTreeClassifier = defaultCopy(extra)
```

Comment:
This was [introduced in 1.5](https://github.com/taishi-oss/spark/commit/43c7ec6384e51105dedf3a53354b6a3732cc27b2#diff-1996e8d94a8a2815261946b240d2507eR90)